src/js/users/components/__tests__/ManageUsers.test.jsx

```diff
@@ -43,7 +43,6 @@ describe("<ManageUsers />", () => {
 
         renderWithRouter(<ManageUsers />, state, history, createReducer);
 
-        expect(await screen.findByLabelText("user-plus")).toBeInTheDocument();
```

Comment:
This should be replaced with something new. This check is needed to ensure the button for creating users appears for admin users